satnogsclient/observer/orbital.py

```diff
@@ -1,56 +1,106 @@
 # -*- coding: utf-8 -*-
-import sys
-
 from datetime import datetime
-
-import ephem
 import pytz
+import ephem
```

Comment:
I never heard before alphabetical ordering is important. In my mind, datetime and pytz go together on grounds of relevance